dom.bs

```diff
@@ -37,6 +37,8 @@ urlPrefix: https://w3c.github.io/hr-time/#; spec: HR-TIME
  type:typedef; urlPrefix: dom-; text: DOMHighResTimeStamp
  type:dfn; text: current high resolution time; url: dfn-current-high-resolution-time
  type:dfn; text: relative high resolution coarse time; url: dfn-relative-high-resolution-coarse-time
+urlPrefix: https://w3c.github.io/trusted-types/dist/spec/#; spec: TRUSTED-TYPES
+ type:dfn; text: [[Data]]; for: TrustedScript; url: trustedscript-data
```

Comment:
This shouldn't be needed? Also why is this using square brackets?

Reply:
We use the internal slot so it's referenced here. Also it uses brackets because that's how the slot is defined?

Reply:
Let me clarify:

1. It should be exported by Trusted Types and since Trusted Types is indexed(?), we shouldn't have to list it explicitly.
2. We generally don't define internal slots this way in web platform specifications and definitely not for any nodes as far as I know.

Reply:
https://html.spec.whatwg.org/#shared-internal-slot:-crossoriginpropertydescriptormap plus many other examples exist in the HTML spec of internal slots being defined with the [[slot]] syntax.

Reply:
You're correct I don't need to define the anchor I think that must have been from before I exported the definition in TT left over erroneously.